docs/plugin-proposal-explicit-resource-management.md

```diff
@@ -0,0 +1,65 @@
+---
+id: babel-plugin-proposal-explicit-resource-management
+title: "@babel/plugin-proposal-explicit-resource-management"
+sidebar_label: explicit-resource-management
+---
+
+This plugin enables Babel to transform using declarations `using handler = await read();`.
+
+## Example
+
+```js title="input.js"
+using handlerSync = openSync();
+await using handlerAsync = await openAsync();
+```
+
+will be transformed to
+
+```js title="output.js"
+try {
+  var _stack = [];
+  var handlerSync = babelHelpers.using(_stack, openSync());
+  var handlerAsync = babelHelpers.using(_stack, await openAsync(), true);
+} catch (_) {
+  var _error = _;
+  var _hasError = true;
+} finally {
+  await babelHelpers.dispose(_stack, _error, _hasError);
+}
+```
+
+[Try it on the REPL](https://babeljs.io/repl/build/54519/#?browsers=ie%2011&build=&builtIns=false&corejs=3.28&spec=false&loose=false&code_lz=K4Zwlgdg5gBAFgQwgEwDYFMBOBlAnhAYxgF4YB7AB3Qj0IAoBKAbgCgEB3BMAFxlEliIUGTAEEQ-IqQ5delauMmMmQA&debug=false&forceAllTransforms=false&modules=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=react&prettier=false&targets=&externalPlugins=%40babel%2Fplugin-proposal-explicit-resource-management%407.22.0%2C%40babel%2Fplugin-external-helpers%407.18.6&assumptions=%7B%7D).
```

Comment:
```suggestion
[Try it on the REPL](https://babeljs.io/repl#?build=&builtIns=false&corejs=3.28&spec=false&loose=false&code_lz=K4Zwlgdg5gBAFgQwgEwDYFMBOBlAnhAYxgF4YB7AB3Qj0IAoBKAbgCgEB3BMAFxlEliIUGTAEEQ-IqQ5delauMmMmQA&debug=false&forceAllTransforms=false&modules=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=react&prettier=false&targets=&externalPlugins=%40babel%2Fplugin-proposal-explicit-resource-management%407.22.0%2C%40babel%2Fplugin-external-helpers%407.18.6&assumptions=%7B%7D).
```

Use the latest `@babel/standalone` build instead of the CI build so that when 7.22.0 is published, the link will be ready to go.